edge-modules/DirectMethodSender/DirectMethodCloudSender.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodSender
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using TransportType = Microsoft.Azure.Devices.TransportType;
+
+    public class DirectMethodCloudSender : DirectMethodSenderBase
+    {
+        readonly ServiceClient serviceClient;
+
+        DirectMethodCloudSender(
+            ServiceClient serviceClient,
+            ILogger logger)
+            : base(
+                logger,
+                Settings.Current.DeviceId,
+                Settings.Current.TargetModuleId)
+        {
+            this.serviceClient = Preconditions.CheckNotNull(serviceClient, nameof(serviceClient));
+            this.serviceClient = serviceClient;
+        }
+
+        public override Task CloseAsync() => this.serviceClient.CloseAsync();
+
+        public static async Task<DirectMethodCloudSender> CreateAsync(
+            string connectionString,
+            TransportType transportType,
+            ILogger logger)
+        {
+            ServiceClient serviceClient = ServiceClient.CreateFromConnectionString(connectionString, transportType);
```

Comment:
you need to dispose service client which implements IDisposable (same for module client in DirectMethodLocalSender).  I suggest you to move Service Client creation in DirectMethodCloudSender and implements IDisposable for DirectMethodCloudSender.